sentry.server.config.ts

```diff
@@ -0,0 +1,21 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://e1dce4791416146de03ff1642ed719d5@o204651.ingest.us.sentry.io/4507788896239616",
```

Comment:
we should move this to env variable so that forks won't use the same dsn

Reply:
done